corehq/apps/reports/standard/deployments.py

```diff
@@ -487,6 +488,40 @@ def _fmt_timestamp(timestamp):
         result[0][1] = table
         return result
 
+    def get_location_column(self, user):
+        assigned_loc_ids = user.get('assigned_location_ids')
+        if not assigned_loc_ids:
+            return '---'
+        primary_loc_id = user.get('location_id')
+        return self._get_formatted_assigned_location_names(primary_loc_id, assigned_loc_ids)
+
+    def _get_formatted_assigned_location_names(self, primary_location_id, assigned_location_ids):
+        """
+        Create an HTML formatted string of the given assigned location names.
+        The primary location will be highlighted in bold.
+        """
+        locs = SQLLocation.objects.filter(location_id__in=assigned_location_ids)
+        formatted_loc_names = []
+        for loc in locs:
+            if loc.location_id == primary_location_id and len(assigned_location_ids) > 1:
```

Comment:
Can't we simply bold the location if its primary irrespective of the length of assigned locations?
It would be confusing if help_text says that primary location is in bold and then for one row its not in bold because its the only location.

Reply:
That's a good point. I thought it might be unnecessary to highlight the location if there is only one, but you make a good point that we should rather keep this consistent given what the help text says. I'll remove the length check from the conditional.

Reply:
[78bb53d](https://github.com/dimagi/commcare-hq/pull/34744/commits/78bb53d4477a313693f62dd4c93e913615ca250b)